ParisBNF/abb/BNFabb137.xml

```diff
@@ -258,21 +289,22 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                 </note>
                              </item>
 
-                             <item xml:id="a2">
+                             <item xml:id="a3">
                                <locus from="2va" to="2vb"/>
                                 <desc type="Unclear" xml:lang="gez"></desc>
                                 <q xml:lang="lat" cert="low">Lagebam Inarya cum mierore stastetie anima MDCCCXLV</q>
```

Comment:
Latin is "la".
I would delete cert="low" and instead mark up the illegible words. I can read "Legebam (I collected) Inarya cum ... ... animi anno MDCCCXLV" It would also be possible to mark up the year.

Reply:
OK.

Reply:
Ok